apps/store/src/components/LayoutWithMenu/LayoutWithMenu.tsx

```
@@ -19,46 +20,52 @@ type LayoutWithMenuProps = {
       className: string
       [GLOBAL_PRODUCT_METADATA_PROP_NAME]: GlobalProductMetadata
       story: PageStory | undefined
+      breadcrumbs?: Array<BreadcrumbListItem>
     }
   >
   overlayMenu?: boolean
   hideFooter?: boolean
 }
 
-export const LayoutWithMenu = ({
-  children,
-  overlayMenu = false,
-  hideFooter = false,
-}: LayoutWithMenuProps) => {
-  const { story, globalStory, className } = children.props
+export const LayoutWithMenu = (props: LayoutWithMenuProps) => {
+  const { story, globalStory, className, breadcrumbs } = props.children.props
 
-  useHydrateProductMetadata(children.props[GLOBAL_PRODUCT_METADATA_PROP_NAME])
+  useHydrateProductMetadata(props.children.props[GLOBAL_PRODUCT_METADATA_PROP_NAME])
   const handleLocaleChange = useChangeLocale(story)
 
   const headerBlock = filterByBlockType(globalStory.content.header, HeaderBlock.blockName)
   const footerBlock = filterByBlockType(globalStory.content.footer, FooterBlock.blockName)
 
-  const showFooter = !hideFooter && !story?.content.hideFooter
+  const showHeader = !story?.content.hideMenu
```

Should we rename it in Storyblok too?

---

hmmm not sure actually, in Storyblok the default is "show" and we have an option to "hide" so I think that's correct

but I like to keep JSX as simple as possible which is why I transform it here - makes sense?